### DISCUSSION TITLE
Browser ignore google congestion controller max bitrate

### DISCUSSION THREAD

**Discussion by asticode** [pion/webrtc#2437] [Q&A]
Hey guys,

I'm using the google congestion controller like this :

```go
i := &interceptor.Registry{}

ccf, _ := cc.NewInterceptor(func() (cc.BandwidthEstimator, error) {
	return gcc.NewSendSideBWE(
		gcc.SendSideBWEInitialBitrate(int(minBitrate)),
		gcc.SendSideBWEMaxBitrate(int(maxBitrate)),
		gcc.SendSideBWEMinBitrate(int(minBitrate)),
	)
})

i.Add(ccf)

webrtc.ConfigureTWCCHeaderExtensionSender(m, i)

webrtc.RegisterDefaultInterceptors(m, i)

api := webrtc.NewAPI(
	webrtc.WithInterceptorRegistry(i),
	webrtc.WithMediaEngine(m),
	webrtc.WithSettingEngine(se),
)
```

I can see that it's making a difference since the browser sent bitrate is way bigger.

Unfortunately, whatever value I set in `maxBitrate`, the browser (Chrome and Firefox) always send around 2,5Mbps (the same is happening if `maxBitrate` is smaller or bigger than 2,5Mbps).

Is there something I'm doing wrong? Or is there anything else I need to do?

Cheers
Quentin

**Comment by silently** [comment]
Hello,

This is a late reply but I think what improved the situation with the browser bitrate is enabling TWCC.

NewSendSideBWE's primary effect is to create an estimator like in this [example](https://github.com/pion/webrtc/blob/2ffab965d3da4e5ab92473964b2de669a3a1518a/examples/bandwidth-estimation-from-disk/main.go#L78), then you can use the estimator to adapt what is sent to the other side of pion.
